docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityStatus.php

```diff
@@ -192,6 +192,28 @@ protected function getPayload(bool $forcePush = FALSE): array {
     return $payload;
   }
 
+  /**
+   * Get operating status details, shortened as necessary.
+   *
+   * @return string
+   *   Details of operating status.
+   */
+  protected function getOperatingStatusMoreInfoShort() : string {
+    if ($this->facilityNode->get('field_operating_status_more_info')->value) {
+      $operatingStatusMoreInfo = $this->facilityNode->get('field_operating_status_more_info')->value;
+      $operatingSatusMoreInfoLength = json_encode($this->facilityNode->get('field_operating_status_more_info')->value);
+      $operatingSatusMoreInfoLength = mb_strlen($operatingSatusMoreInfoLength);
```

Comment:
This one bit is deceiving, At this point it is not a length. yet we are calling it a lenght.  Feels like it should be something else.  Like this?
```suggestion
      $operatingSatusMoreInfoJson = json_encode($this->facilityNode->get('field_operating_status_more_info')->value);
      $operatingSatusMoreInfoLength = mb_strlen($operatingSatusMoreInfoJson);
```